A/AquaCrop/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+using BinaryBuilder, Pkg
+
+name = "AquaCrop"
+version = v"7.1"
+
+# url = "https://github.com/KUL-RSDA/AquaCrop"
+# description = "FAO AquaCrop model of plant growth"
+
+sources = [
+    GitSource("https://github.com/KUL-RSDA/AquaCrop",
+              "33ae89706ff82c2c119930e47899cd3fad519d6f"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/AquaCrop*/src/
+
+sed -i -e 's/-march=native//g' Makefile
+
+export FC
+make -j${nproc}
+
+# on x86_64-w64-mingw32-libgfortran4, the binary is 'aquacrop'
+# on i686-w64-mingw32-libgfortran5, the binary is 'aquacrop.exe'
+if [ -f aquacrop.exe ]; then
+    mv aquacrop.exe aquacrop
+fi
+install -Dvm 755 aquacrop "${bindir}"/aquacrop${exeext}
```

Comment:
```suggestion
install -Dvm 755 "aquacrop${exeext}" -t "${bindir}"
```
Side note, I don't understand why the variable expansion was outside of the quotes.